docs/docker-images.adoc

```diff
@@ -29,19 +29,14 @@ The three initialization images are:
 |===
 | Image | Description
 
-| `docker.inetsoft.com/inetsoft/init-db`
-| Initializes a database data space.
+| `docker.inetsoft.com/inetsoft/init`
+| Initializes the data space.
 
-| `docker.inetsoft.com/inetsoft/init-fs`
-| Initializes a default file system data space.
-
-| `docker.inetsoft.com/inetsoft/init-oak`
-| Initializes a new data space based on   http://jackrabbit.apache.org/oak/[Apache Jackrabbit Oak] that is optimized for use in Kubernetes.
 |===
 
-All initialization images _must_ mount a volume at `/var/lib/inetsoft/shared`. This volume must be shared with the server and scheduler containers.
+The initialization image _must_ mount a volume at `/var/lib/inetsoft/shared`. This volume must be shared with the server and scheduler containers.
 
-NOTE: It is strongly recommended that the Oak data space is used in Kubernetes or a clustered deployment in Docker.
+NOTE: It is strongly recommended that the data space is used in Kubernetes or a clustered deployment in Docker.
```

Comment:
I don't know what is trying to be communicated here, I think this statement can be removed.